packages/shared_preferences/shared_preferences/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.2
```

Comment:
nit: I'd recommend bumping to 2.4.0 since adding a new tool for the package seems like it should be more than a patch release. @stuartmorgan wdyt with respect to versioning?

Reply:
Yes, new functionality should be a minor version.